packages/rune-games-cli/src/flows/Create.tsx

```diff
@@ -65,6 +65,21 @@ function emptyDir(dir: string) {
   }
 }
 
+const isRelativePath = (path: string) =>
+  path.startsWith(".") || path.startsWith("..")
+
+const getPackageName = (targetDir: string) => {
+  if (!isRelativePath(targetDir)) return targetDir
```

Comment:
Yeah I guess we can add a ternary operator where pkg.name is assigned. I thought encapsulating all the package name logic into one function is more readable.

I accounted for directory name containing spaces or other symbols by replacing non-hyphen characters with hyphens. I didn't account for when the user types in weird characters for targetDir though

Reply:
Sorry, I meant specifically removing the line 72, so that your logic is always used. That way we'll always make sure to use a valid name for package.json

Reply:
Ah yep. I updated it so that it transforms it to valid package name and only performs directory operations if it's a relative path